File: lua/1012.lua
local A, B, C = io.read("*n", "*n", "*n")
print("TRIANGULO: " .. string.format("%0.3f", (0.5*A*C)));
print("CIRCULO: " .. string.format("%0.3f", (3.14159*C*C)));
print("TRAPEZIO: " .. string.format("%0.3f", (A+B)*C/2));
print("QUADRADO: " .. string.format("%0.3f", B*B));
print("RETANGULO: " .. string.format("%0.3f", A*B));

File: c/1012.c
#include <stdio.h>

int main()
{
    double A, B, C;
    scanf("%lf %lf %lf", &A, &B, &C);

    printf("TRIANGULO: %0.3lf\n", (0.5*A*C));
    printf("CIRCULO: %0.3lf\n", (3.14159*C*C));
    printf("TRAPEZIO: %0.3lf\n", (A+B)*C/2);
    printf("QUADRADO: %0.3lf\n", B*B);
    printf("RETANGULO: %0.3lf\n", A*B);

	return 0;
}

